packages/playground/assets/css/css-url.css

```diff
@@ -26,6 +26,29 @@
   background-size: 10px;
 }
 
+.css-image-set-with-var {
+  --bg-img: url('../nested/asset.png');
+  background-image: -webkit-image-set(var(--bg-img) 1x, var(--bg-img) 2x);
+  background-size: 10px;
+}
+
+.css-image-set-mix-url-var {
+  --bg-img: url('../nested/asset.png');
+  background-image: -webkit-image-set(
+    var(--bg-img) 1x,
+    url('../nested/asset.png') 2x
+  );
+  background-size: 10px;
+}
+
+.css-image-set-multiple-descriptor {
+  background-image: -webkit-image-set(
+    '../nested/asset.png' type('image/png') 1x,
+    '../nested/asset.png' type('image/png') 2x
+  );
+  background-size: 10px;
+}
```

Comment:
I think this needs to test inline CSS in HTML because it doesn't have `url(` and also needs to compile, may have had a problem with inline CSS in HTML.

Reply:
Would you share a failing example to add to the PR?

Reply:
https://github.com/vitejs/vite/blob/8e1d6ae0b00207ab8fd6e1f0e0c1ec73cab5c1f5/packages/vite/src/node/plugins/html.ts#L359-L365 in here.

Reply:
because I no add the assertion. 😂

Reply:
@poyoho gotcha. Should the fix for this case be part of this PR, though? This wasn't working before even without `var`. I think we could merge this PR and then create a new PR or issue for inline style support, no?

Reply:
OK, I think this issue had two solutions. one is to remove the judge. other is add the judge. remove this judge may be better. But anyway it is also not part of this PR. (it seems to part of html instead of css).